docs/guides/how_to_guides/configuring_datasources/how_to_configure_sorting_in_data_connectors.rst

```diff
@@ -0,0 +1,1029 @@
+.. _how_to_guides__how_to_sort_batches:
+
+How to Configure a Data Connector to Sort Batches
+==================================================
+
+.. warning::
+
+  Sorters are configured through a Data Connector, which is a concept that was introduced in the V3 (BatchRequest) API.
+  The following configuration is not supported by the V2 (batch_kwargs) API.
+
+This guide demonstrates how to sort :ref:`batches <reference__core_concepts__glossary__datasources>` of data that Great Expectations can validate from a filesystem using a configured ``DataConnector``.
+To read more about DataConnectors please refer to the doc: :ref:`How to choose which DataConnector to use <which_data_connector_to_use>`.   For this how-to-guide, we will be using a
+``ConfiguredAssetFilesystemDataConnector`` as an example.
+
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
+
+    - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
+    - :ref:`Understand the basics of Datasources in 0.13 or later <reference__core_concepts__datasources>`
+    - Learned how to configure a :ref:`DataContext using test_yaml_config <how_to_guides_how_to_configure_datacontext_components_using_test_yaml_config>`
+    - Learned how to :ref:`create a Batch <how_to_guides__creating_batches>`
+
+
+Contents
+---------
+- :ref:`Configuring a Lexicographical Sorter <how_to_guides__how_to_sort_batches_LexicographicalSorter>`
+- :ref:`Configuring a Numeric Sorter <how_to_guides__how_to_sort_batches_NumericSorter>`
+- :ref:`Configuring a Datetime Sorter <how_to_guides__how_to_sort_batches_DatetimeSorter>`
+- :ref:`Configuring a CustomList Sorter <how_to_guides__how_to_sort_batches_CustomListSorter>`
+- :ref:`Configuring Multiple Sorters <how_to_guides__how_to_sort_batches_MultipleSorters>`
+- :ref:`Configuring Sorters with Custom Filters <how_to_guides__how_to_sort_batches_SortersWithFilters>`
+
+.. _how_to_guides__how_to_sort_batches_LexicographicalSorter:
+
+Configuring a Lexicographical Sorter
+------------------------------------
+
+If you have the following ``lexicographic_example/`` directory in your filesystem, and you want to treat ``*.csv`` files as batches within the ``my_data_asset`` DataAsset:
+
+**Note** : In our example, the ``base_directory`` is set to ``../``. If we are running this Notebook in the same folder as Great Expectations home directory (ie ``great_expectations/``), GE will begin looking for the files in the parent directory.
```

Comment:
I think we should move this note below the code block bash code block that is currently below.

It might also be good to add a line in this note about how you can also configure a base directory with an absolute path, which we do in the below examples.

Reply:
done :)